src/lib.rs

```diff
@@ -263,21 +332,11 @@ pub fn read_v1_raw_message<R: Read>(
         }
     }
 
-    let mut message = MAVLinkV1MessageRaw {
-        start: MAV_STX,
-        payload_length: reader.read_u8()?,
-        sequence: reader.read_u8()?,
-        system_id: reader.read_u8()?,
-        component_id: reader.read_u8()?,
-        message_id: reader.read_u8()?,
-        payload_buffer: [0; 255],
-        checksum: 0,
-    };
-
-    let payload = &mut message.payload_buffer[..message.payload_length as usize];
-    reader.read_exact(payload)?;
+    let mut message = MAVLinkV1MessageRaw::new();
```

Comment:
```suggestion
    let mut message = Self::new();
```

Reply:
The `read_v1_raw_message` function is not method. I could not apply `Self` here.

Reply:
👍